apps/omg_child_chain_rpc/test/omg_child_chain_rpc/web/controllers/transaction_test.exs

```diff
@@ -17,8 +17,27 @@ defmodule OMG.ChildChainRPC.Web.Controller.TransactionTest do
   use ExUnit.Case, async: false
 
   alias OMG.ChildChainRPC.Web.TestHelper
+  @table_name :alarms
+  setup_all do
+    {:ok, pid} = OMG.ChildChainRPC.Application.start([], [])
+    _ = Application.load(:omg_child_chain_rpc)
+    table_setup()
+
+    on_exit(fn ->
+      ref = Process.monitor(pid)
+
+      receive do
+        {:DOWN, ^ref, :process, _, _} ->
+          # a tiny wait to allow the endpoint to be brought down for good, not sure how to get rid of the sleep
+          # without it one might get `eaddrinuse`
+          Process.sleep(10)
+          :ok
+      end
```

Comment:
Maybe we should have them in TestHelper then?

Reply:
The same suggestion as to https://github.com/omisego/elixir-omg/pull/1382/files#diff-44f1ab34a5d784c25fcdf593b3a3e31f applies